declarations/Linear.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "Linear",
+  "documents": {
+    "Terms of Service": {
+       "fetch": "https://linear.app/terms",
```

As reported by the linter: there is whitespace problem on that line.
```suggestion
    "fetch": "https://linear.app/terms",
```
If you edit JSON files manually, you can run the following command within your declarations repository to format the files: `npm run lint:fix` 🙂